homework-objects.js

```diff
@@ -0,0 +1,80 @@
+// 1 Задача про обчислення різниці часу
+
+function periodOfTime(
+  startDate = '01 Jan 1990',
+  endDate = '15 Mar 2023',
+  timeCounter = 'hours'
+) {
+  if (timeCounter === 'days') {
+    const diffInDays =
+      (Date.parse(startDate) - Date.parse(endDate)) / 1000 / 86400;
+    return Math.abs(diffInDays);
+  }
+  if (timeCounter === 'hours') {
+    const diffInHours =
+      (Date.parse(startDate) - Date.parse(endDate)) / 1000 / 3600;
+    return Math.abs(diffInHours);
+  }
+  if (timeCounter === 'minutes') {
+    const diffInMinutes =
+      (Date.parse(startDate) - Date.parse(endDate)) / 1000 / 60;
+    return Math.abs(diffInMinutes);
+  }
+  if (timeCounter === 'seconds') {
+    const diffInSeconds = (Date.parse(startDate) - Date.parse(endDate)) / 1000;
+    return Math.abs(diffInSeconds);
```

Comment:
There is a lot of copy/paste. Do you see it? Let's fix it.
1. Create a separate variables for `Date.parse(startDate)` and `Date.parse(endDate)` 
2. Let's create and use an object with keys - timeCounter and value - numbers